docs/source/udf/advanced.rst

```diff
@@ -276,6 +276,7 @@ available as properties of the :attr:`libertem.udf.UDF.meta` attribute of type
 Common applications include allocating buffers with a :code:`dtype` or shape
```

Comment:
In this file, there are still some references to the old location of the `MaskContainer`